extra/1-currency-conversion.js

```diff
@@ -15,13 +15,17 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(num) 
+{
+      num = num*99/100*5.7;
```

Comment:
Again, please make sure that this is readable, with correct formatting and indentation.

It's even more readable to put the numbers into variable when necessary so if we need to update it the next time, it's easier to spot which parts need changing, e.g.
```
let rateToBrl = 5.7;
let fee = 99/100;
num = num * fee * rateToBrl;
```
and so on... do you see now how this is a more readable code?